src/scrabble.js

```diff
@@ -1,5 +1,64 @@
-function scrabble() {
+const scores = {
+  a: 1,
+  e: 1,
+  i: 1,
+  o: 1,
+  u: 1,
+  l: 1,
+  n: 1,
+  r: 1,
+  s: 1,
+  t: 1,
+  d: 2,
+  g: 2,
+  b: 3,
+  c: 3,
+  m: 3,
+  p: 3,
+  f: 4,
+  h: 4,
+  v: 4,
+  w: 4,
+  y: 4,
+  k: 5,
+  j: 8,
+  x: 8,
+  q: 10,
+  z: 10
+}
+
+const notValid = 'Input not valid:'
+const validWord = 'Great, your point is:'
+const singleValue = scores
+
+function scrabble(letter) {
+  if (typeof letter !== 'string') {
+    return `${notValid} 0`
+  } else if (letter === '!' || letter === '.') {
+    return `${notValid} 0`
+  } else if (letter === ' ' || letter === '\t\n') {
+    return `${notValid} 0`
```

Comment:
Your code is very easy to read and is very understandable the potential is there for this to be a really strong process. Great job!